backend/README.md

```diff
@@ -17,7 +64,6 @@ Note: `yarn global add` will default to adding packages to `~/.yarn/bin` so by d
 1.  You can add yarn's default installation location to your `$PATH` environment variable, so that your terminal knows where to look for the executables. You can do this by adding `export PATH="$(yarn global bin):$PATH"` as a line in your `~/.bashrc`. Then simply quit your terminal and reopen it (or run source ~/.bashrc) and everything should work.
 2.  Alternatively, you can change the default global location for yarn (`yarn config set prefix <filepath>`) and set `filepath` to something like `/usr/local` or `/usr` (or anything else in your `$PATH` that makes sense). This second method will require you to re-run the above global add command.
 
-Then, you must copy a `.env` file and a `keys.json` file to this folder.
```

Comment:
why did we remove this line?

Reply:
Good question — I just factored it out of that portion and moved it to the top since it’s required to run both manually and with Docker. It’s moved to the top of the file.

Reply:
ah sorry, forgot about that by the time I got down here, my bad. Looks good to go then.